src/conditionals/simple_programs_with_decision_points/hard/q8/MainTest.java

```diff
@@ -0,0 +1,63 @@
+package conditionals.simple_programs_with_decision_points.hard.q8;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+
+    static final int THRESHOLD = 50;
+
+    @Override
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "1\n2";
+
+        return new Clause[]{
+            new StringLiteral("Enter x: "),
```

Comment:
if this is Java, get rid of the extra spaces here because that's what the optionals are for, if it's parsons you don't need the optionals